app/src/main/java/fr/free/nrw/commons/explore/SearchActivity.java

```diff
@@ -211,49 +211,32 @@ public void onMediaClicked(int index) {
                     .beginTransaction()
                     .hide(supportFragmentManager.getFragments().get(supportFragmentManager.getBackStackEntryCount()))
                     .add(R.id.mediaContainer, mediaDetails)
-                    .addToBackStack(null)
                     .commit();
-            // Reason for using hide, add instead of replace is to maintain scroll position after
+            // Reason for using hide, add instead of replace is to maintain scroll0 position after
             // coming back to the search activity. See https://github.com/commons-app/apps-android-commons/issues/1631
             // https://stackoverflow.com/questions/11353075/how-can-i-maintain-fragment-state-when-added-to-the-back-stack/19022550#19022550
             supportFragmentManager.executePendingTransactions();
         }
         mediaDetails.showImage(index);
     }
 
-    /**
-     * This method is called on Screen Rotation
-     */
-    @Override
-    protected void onResume() {
-        if (supportFragmentManager.getBackStackEntryCount()==1){
-            //FIXME: Temporary fix for screen rotation inside media details. If we don't call onBackPressed then fragment stack is increasing every time.
```

Comment:
This code seems like a temporary fix for orientation change, why did you remove this one?

Reply:
we check above  condition when we  added fragment in   supportFragementManager backstack
but we are not add any fragement in fragementManager backstack
To add fragement in fragementManager backstack we use `supportFragementManager.addToBackStack(null)
`

`

Reply:
Sorry, I couldn't understand your explanation. Can you explain it again?